Three/include/CGAL/Three/Scene_interface.h

```diff
@@ -45,13 +45,13 @@ class Scene_group_item;
  */
 enum RenderingMode { Points = 0,
                      PointsPlusNormals,
+                     ShadedPoints,
                      Splatting,
                      Wireframe, 
                      Flat,
                      FlatPlusEdges,
                      Gouraud,
-                     LastRenderingMode = Gouraud,
-                     NumberOfRenderingMode = LastRenderingMode+1 };
+                     NumberOfRenderingMode};
```

Comment:
That will break the binary compatibility: if you add `ShadedPoints` at the end of the enum, before `NumberOfRenderingMode`, then only the binaries using `NumberOfRenderingMode` will have to be rebuild. Otherwise, any item of plugin that uses render modes different from `Points` or `PointPlusNormals`, will have to be recompiled.